nemo/collections/nlp/models/language_modeling/megatron/bert_model.py

```diff
@@ -242,6 +250,8 @@ def __init__(
             init_method=init_method_normal(init_method_std), vocab_size=vocab_size, hidden_size=hidden_size
         )
 
+        if skip_head:
+            self.post_process = False
         if self.post_process:
```

Comment:
if not skip_head and if self.post_process :  ?? Or maybe if self.post_process is used in other areas just leave it as such.

Reply:
yeah, it affect other places as well. But I can change it all to `if not skip_head and if self.post_process`. Do you want me to change? I remember current way is cleaner in terms of code but not necessarily more readable

Reply:
Well, if you think its cleaner then leave it as such.